src/containers/SinglePost.js

```diff
@@ -141,7 +142,7 @@ const setupPage = (store, id, query, action) => {
   }
 
   return Promise.all([
-    saveCurrentCommunityId(dispatch, communityId, userId),
+    saveCurrentCommunity(dispatch, community, userId),
```

Comment:
This was left over from my first approach, where I needed the community name in this function. Fixed now.